src/phantasm-renderer/common/hashable_storage.hh

```diff
@@ -30,8 +30,13 @@ public:
     hashable_storage(hashable_storage const& rhs) { std::memcpy(&_storage.value, &rhs._storage.value, sizeof(T)); }
     hashable_storage& operator=(hashable_storage const& rhs) { std::memcpy(&_storage.value, &rhs._storage.value, sizeof(T)); }
 
-    hashable_storage(hashable_storage&&) = delete;
-    hashable_storage& operator=(hashable_storage&&) = delete;
+    hashable_storage(hashable_storage&& rhs) noexcept { std::memcpy(&_storage, &rhs._storage, sizeof(_storage)); }
+    hashable_storage& operator=(hashable_storage&& rhs) noexcept
+    {
+        if (this != &rhs)
+            std::memcpy(&_storage, &rhs._storage, sizeof(_storage));
+        return *this;
+    }
```

Comment:
you can just delete these. this type cannot be moved faster than it can be copied, so `const&` is sufficient.

Reply:
right, fixed!